01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+numbers = (1..20).to_a
+numbers.each{|number| 
```

Comment:
```suggestion
(1..20).each { |number| 
```

to_aせずにそのままeachでループさせて大丈夫です。
あと`{`の前後には空白を入れることが多いですね。
ただ、今回は複数行のブロックなので慣習的に`{}`よりも`do/end`を使うことが多いです。
ここもdo/endに書き直してみてください〜。

Reply:
レビューありがとうございます！
> to_aせずにそのままeachでループさせて大丈夫です。

ターミナルで実際に動かして`numbers = (1..20)`を確認してみました。承知いたしました。

> 今回は複数行のブロックなので慣習的に{}よりもdo/endを使うことが多いです。

{}を使うときは確か、処理が１行で収まるときでしたね。チェリー本で復習します。